### PR TITLE
VW: add missing FW for Skoda Fabia

### PR DESCRIPTION
**Pull request by relo1**
Added my Skoda Fabia

### CODE REVIEW

**Comment by sshane** [comment]
What's your dongle ID?

**Comment by relo1** [comment]
2207663391ae459c

**Comment by sshane** [comment]
The repo is archived, can you unarchive it?

File: selfdrive/car/volkswagen/values.py [hunk 1]
```diff
@@ -1160,18 +1160,24 @@ def init_make(self, CP: car.CarParams):
   CAR.SKODA_FABIA_MK4: {
     (Ecu.engine, 0x7e0, None): [
       b'\xf1\x8705E906018CF\xf1\x891905',
+      b'\xf1\x8705C906032L \xf1\x891701',
+      b'\xf1\x8705C906032L ',
```
Comment (sshane) [inline selfdrive/car/volkswagen/values.py:1164]:
```suggestion
      b'\xf1\x8705C906032L \xf1\x891701',
```

**Review by thegauravgiri — APPROVED** [review]
LGTM

**Comment by relo1** [comment]
please excuse, i'm bloody new here...
do I have to pull again?

**Comment by sshane** [comment]
You deleted the repo, so GH won't allow to re-open, yes

**Pull request closed without merging**